_locales/cs/maqueen-strings.json

```diff
@@ -0,0 +1,128 @@
+{
+  "Maqueen_V5.BatteryType.Alkaline|block": "alkalické",
+  "Maqueen_V5.BatteryType.Lithium|block": "lithiové",
+  "Maqueen_V5.BleCmd.BleBackward|block": "jeď dozadu",
+  "Maqueen_V5.BleCmd.BleForward|block": "jeď dopředu",
+  "Maqueen_V5.BleCmd.BleLeft|block": "zadboč vlevo",
+  "Maqueen_V5.BleCmd.BleRgbB|block": "RGB modrá",
+  "Maqueen_V5.BleCmd.BleRgbGB|block": "RGB azurová",
+  "Maqueen_V5.BleCmd.BleRgbG|block": "RGB zelený",
+  "Maqueen_V5.BleCmd.BleRgbOff|block": "RGB vypnuty",
+  "Maqueen_V5.BleCmd.BleRgbRB|block": "RGB fialová",
+  "Maqueen_V5.BleCmd.BleRgbRGB|block": "RGB bílá",
+  "Maqueen_V5.BleCmd.BleRgbRG|block": "RGB žlutá",
+  "Maqueen_V5.BleCmd.BleRgbR|block": "RGB červená",
+  "Maqueen_V5.BleCmd.BleRight|block": "zaboč vpravo",
+  "Maqueen_V5.BleCmd.BleServo1Left|block": "otoč servo1 doleva",
+  "Maqueen_V5.BleCmd.BleServo1Right|block": "otoč servo1 doprava",
+  "Maqueen_V5.BleCmd.BleServo2Left|block": "otoč servo2 doleva",
+  "Maqueen_V5.BleCmd.BleServo2Right|block": "otoč servo2 doprava",
+  "Maqueen_V5.BleCmdDefault|block": "proveď Bluetooth příkaz %cmd=bleCommand(jeď dopředu)",
+  "Maqueen_V5.BleGetCmd|block": "poslední Bluetooth příkaz",
+  "Maqueen_V5.BlecallbackUser|block": "při přijetí Bluetooth příkazu",
+  "Maqueen_V5.CarLightColors.Black|block": "černou",
+  "Maqueen_V5.CarLightColors.Blue|block": "modrou",
+  "Maqueen_V5.CarLightColors.Cyan|block": "azurovou",
+  "Maqueen_V5.CarLightColors.Green|block": "zelenou",
+  "Maqueen_V5.CarLightColors.Purple|block": "fialovou",
```

Comment:
I don't have the maqueen v5 car to judge the actual colour, but purple is usually translated as purpurová. The "fialová" represents a different colour.

Considering the context where the NeoPixelColors.Purple defines it as red and blue mix (0xFF00FF), I'd say purpurová is proper translation.